### DISCUSSION TITLE
771

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#941] [Comments]
# 771

온보딩의 반대말로 오프보딩이 있다. 조직을 나가는 직원에 대한 퇴사 절차를 의미한다. 보통 퇴사 이후에도 회사에 대한 좋은 이미지를 남겨주기 위해 좋은 방향으로 퇴사절차를 밟는다. 여기서 중요한 것은 절차에 집중한다는 것이다. 이런 절차도 물론 중요하겠지만, 그 팀원이 정말 좋은 팀원이였다면 떠나는 이가 미안해 하지 않도록 하는 것이 중요하다. 그리고 이를 위해 떠난 이후에도 조직이 계속해서 더 잘되는 모습을 보여줘야한다. 소라의 날개 31권에서도 비슷한 이야기가 나오는데 많이 공감되었다. (오른쪽에서 왼쪽방향으로 읽어야 한다.) 떠나는 팀원이나 남은 팀원이나 서로에게 소중한 시간을 함께한 동료였다면 부정적인 감정이 전파되길 바라지는 않을 것이다. 만약 내가 떠나고 동료들이 더 힘들어하거나 망하길 바란다..

https://jojoldu.tistory.com/771

**Comment by AppleCEO** [comment]
떠나간 자기 자신도 더 잘 되도록 노력해야겠네요. 좋은 글 감사합니다~

**Comment by Hoo93** [comment]
마침 3월 초부터 퇴사 절차를 밟고 있었는데 여러 생각들이 드는 글이네요.
항상 좋은 글 감사합니다 !
